package.json

```diff
@@ -8,7 +8,16 @@
     "start": "react-native start",
     "test": "jest",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
-    "clean-up": "rm -rf .git README.md && rm -rf ./assets/RN-Typescript-Boilerplate.gif ./assets/logo.png && git init"
+    "clean-up": "rm -rf .git README.md && rm -rf ./assets/RN-Typescript-Boilerplate.gif ./assets/logo.png && git init",
+    "postinstall": "jetify",
+    "ios11": "react-native run-ios --simulator=\"iPhone 11\"",
+    "ios8": "react-native run-ios --simulator=\"iPhone 8\"",
+    "ios8p": "react-native run-ios --simulator=\"iPhone 8 Plus\"",
+    "ios11promax": "react-native run-ios --simulator=\"iPhone 11 Pro Max\"",
```

Comment:
We do not need these, there are too specific for the dev to dev. Devs can add these as optional for themselves.

Reply:
Will remove this.